src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/GitContentManager.java.rej

```diff
@@ -0,0 +1,21 @@
+diff a/src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/GitContentManager.java b/src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/GitContentManager.java	(rejected hunks)
```

Comment:
Erm, this shouldn't have been committed! Although be sure that whatever patch you tried to apply did get applied fully.

Reply:
Haha oops! I did do some unusual git switching and reverting between implementations. I'll remove this.